test/internet/test-http-dns-fail.js

```diff
@@ -4,7 +4,6 @@
  * should trigger the error event after each attempt.
  */
 
-var common = require('../common');
```

Comment:
If it's just this leaked global check, I see two options:

- we remove that leaked global check. these can be warned through the linter mostly, unless we do something like global.x, but that ought to be intentional.
- we preload common.js